app/src/main/java/com/kms/cura/utils/DataUtils.java

```diff
@@ -113,4 +126,16 @@ public static void loadProfile(UserEntity user, ImageView profile, Context conte
         }
     }
 
+    public static List<AppointmentEntity> getDoctorAppt(List<AppointmentEntity> appts){
```

Comment:
Also, should rename this method to make it clearer ( like what kind of appointment this method return )